### ISSUE TITLE
Babbel ESM Compiling issue

### ISSUE THREAD

**Issue by baoduy** [mui-org/material-ui-pickers#1225]
Hi All,
After upgrading to the latest version of material-iu-picker I got below issue. Could you please help to let me know how to fix this?

```cmd
 Build error occurred
/.../node_modules/@babel/runtime/helpers/esm/extends.js:1
export default function _extends() {
^^^^^^

SyntaxError: Unexpected token export
    at Module._compile (internal/modules/cjs/loader.js:721:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/Users/steven/_Code/CodesERP/CoreErp.Portal/ClientApp/node_modules/@material-ui/pickers/dist/material-ui-pickers.js:7:32)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.KNS6 (/Users/steven/_Code/CodesERP/CoreErp.Portal/ClientApp/dist/server/static/NHt7qFJ6icCMMwrcZnODn/pages/chartOfAccount.js:1:80009)
npm ERR! code ELIFECYCLE
```

**Comment by denyo** [comment]
Dupe of #1219.

**Comment by baoduy** [comment]
@denyo I saw you close 1219 without any workaround or solution provided. 

Meanwhile, I downgraded to 3.2.0 when awaiting a fix for 3.2.1

Thanks

**Comment by dmtrKovalenko** [comment]
Released in v3.2.2
